service/src/test/java/bio/terra/workspace/service/workspace/flight/CreateGcpContextFlightV2Test.java

```diff
@@ -143,29 +142,6 @@ void successCreatesProjectAndContext() throws Exception {
     assertPolicyGroupsSynced(workspaceUuid, project);
   }
 
-  @Test
-  @DisabledIfEnvironmentVariable(named = "TEST_ENV", matches = BUFFER_SERVICE_DISABLED_ENVS_REG_EX)
-  void createsProjectAndContext_noBillingAccount_flightFailsAndGcpProjectNotCreated()
```

Comment:
I do not believe this test asserts a realistic use case anymore. Given the validation code in the compact constructor for SpendProfile, it should not be possible to instantiate a GCP spend profile that has no billing account ID:
* If there is a bad profile in the config such as this, the system will not boot while wiring up the config fixture data.
* If there is a bad profile coming back from BPM, we will fail to instantiate the SpendProfile model and raise an exception